.github/workflows/keep-alive.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         touch _pass/pass.txt
 
     - name: Keep workflow alive
-      uses: peaceiris/actions-gh-pages@v3
+      uses: peaceiris/actions-gh-pages@373f7f263a76c20808c831209c920827a82a2847 #v3.9.3
```

Comment:
This will get updated by `.github/dependabot.yml` along with the rest of the GitHub Actions used in these workflows.